components/brave_rewards/browser/BUILD.gn

```diff
@@ -14,6 +15,14 @@ static_library("browser") {
     "BITFLYER_STAGING_CLIENT_ID=\"$bitflyer_staging_client_id\"",
     "BITFLYER_STAGING_CLIENT_SECRET=\"$bitflyer_staging_client_secret\"",
     "BITFLYER_STAGING_URL=\"$bitflyer_staging_url\"",
+    "GEMINI_API_URL=\"$gemini_api_url\"",
```

Comment:
these should not be duplicated like this. Normally you would use a config for this, but these are never even accessed in components/brave_rewards so they shouldn't be here at all.

Reply:
updated - https://github.com/brave/brave-core/pull/8566/files#diff-41756d127f52e09eaa4d10553f84a5a4f6b8fd674c623ec67015a367e4f79f93R82